src/main/kotlin/no/nav/fo/forenkletdeploy/service/TeamService.kt

```diff
@@ -46,39 +47,43 @@ class FOTeam: ITeam(
         environments = listOf("Q6", "Q0", "P")
 )
 
-class TeamSoknad: ITeam(
+class TeamSoknad : Team(
         id = "sd",
         displayName = "Team søknad",
         configUrl = "https://raw.githubusercontent.com/navikt/jenkins-dsl-scripts/master/team_soknad/config.json",
         jenkinsFolder = "team_soknad",
-        extraApps = arrayListOf(
-                ApplicationConfig( name = "soknad-kontantstotte", gitUrl = "https://github.com/navikt/soknad-kontantstotte.git" ),
-                ApplicationConfig( name = "soknad-aap-utland", gitUrl = "https://github.com/navikt/soknad-aap-utland.git" ),
-                ApplicationConfig( name = "soknad-kontantstotte-api", gitUrl = "https://github.com/navikt/soknad-kontantstotte-api.git" ),
-                ApplicationConfig( name = "soknad-html-generator", gitUrl = "https://github.com/navikt/soknad-html-generator.git" ),
-                ApplicationConfig( name = "soknad-kontantstotte-proxy", gitUrl = "https://github.com/navikt/soknad-kontantstotte-proxy.git" )
-        ),
         environments = listOf("T6", "Q0", "P")
-)
+) {
+    init {
+        extraApps = arrayListOf(
```

Comment:
Hvorfor endrer dette seg egentlig?

Reply:
For å legge inn team i appene så måtte teamet være opprettet før jeg kunne opprette `extraApps`, så da måtte `extraApps` bli en `var`, og da er det best å initialisere verdien sånn 👇

Reply:
👍